.github/workflows/build-docs.yml

```diff
@@ -10,11 +10,52 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
+      - name: setup python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        env:
+          GHREPO: "pylbiczi"
+          REPO_PATH: "https://${{secret.ACCESS_TOKEN}}@github.com/${{GHREPO}}.git"
+          run: |
+            python -m pip install --upgrade pip
+            pip install sphinx
+            pip install sphinx-rtd-theme
+            export GHPAGES=[ "$(git ls-remote --heads "$REPOSITORY_PATH" "gh-pages" | wc -l)" -eq 0 ]
+          shell: bash
+        - name: Create gh-pages
+          if: $GHPAGES
+          run: |
+            echo "Creating remote branch ${BRANCH} as it doesn't exist..."
+            git checkout "${BASE_BRANCH:-master}"
+            git checkout --orphan $BRANCH
+            git rm -rf .
+            touch README.md
+            git add README.md
+            git commit -m "Initial ${BRANCH} commit"
+            git push $REPOSITORY_PATH $BRANCH
```

Comment:
Because parsing git commands are hard. I assume this is the "Hey if gh-pages branch doesn't exist, make it" command?